core/arch/arm/kernel/generic_boot.c

```diff
@@ -803,37 +872,51 @@ static void init_fdt(unsigned long phys_fdt)
 	if (!fdt)
 		panic();
 
+	dt->blob = fdt;
+
+	/*
+	 * The model here is either:
+	 * 1. Pass a DTB overlay from a previous boot stage for appendation
+	 * 2. Pass no DTB overlay. An overlay tree will be created @ CFG_DT_ADDR
+	 * If CFG_EXTERNAL_DTB_OVERLAY is not defined the call to
+	 * init_dt_overlay is a nop
+	 */
```

Comment:
this is also  described in the **documentation/optee_design.md** and in **config.mk**.
Should better describe at one place.

Reply:
I'll zap the comment in the code.